### ISSUE TITLE
Add a 1080p50 mode

### ISSUE THREAD

**Issue by philipstarkey** [chriz2600/time-sleuth#1]
Hi,

I was wondering if it was possible to add a 1080p50 mode or if there is a hardware limitation that prevents that?

I'm interested in using this device to quantify lag in live broadcasts but all of our equipment runs at 50Hz (to match the powerline frequency in our country).

**Comment by chriz2600** [comment]
Good idea, I will add 50Hz modes (1080p50, 1080i50, 720p50) as options to the custom firmware generator at time-sleuth.i74.de.

**Comment by chriz2600** [comment]
I just added 1080p50, 1080i50 and 720p50 as options to [time-sleuth.i74.de](http://time-sleuth.i74.de). To flash your custom firmware you need need the tools listed [here.](https://github.com/chriz2600/time-sleuth/tree/experimental#firmware)
If you need help on programming the firmware, feel free to contact me.

**Comment by philipstarkey** [comment]
Amazing, thank you!

I've ordered the device and I'll let you know once I get a chance to test out the new firmware

**Comment by philipstarkey** [comment]
I had a chance to test it today. For the most part it works as expected. The video mixers typically require very strict (consistent) input formats (resolution and framerate). When I directly connect the time sleuth (outputting 1080p50) to the video mixer, I see the signal for a split second before it vanishes (and it doesn't seem to fill the screen completely on the monitor output). However, if I feed the time sleuth in via a [Decimator converter](https://www.decimator.com/Products/MiniConverters/MD-HX/MD-HX.html), then it does work as expected. Interestingly, the converter detects the input as 1080p50 as expected.

There are a couple of possibilities off the top of my head - one is that sometimes the video mixer can be finicky about the EDID, and the decimator rewrites this. Another is that the timing of the HDMI output could be off slightly, which is why it works for a split second and then vanishes. I'm by no means an expert on the finer details of the HDMI protocol though, so you probably know more than me.

Up to you whether you want me to investigate this further or not (since I can make it work with the Decimator converter). I could switch everything over to 1080p60 temporarily to see if the problem still exists if that is helpful.

**Comment by chriz2600** [comment]
Time Sleuth does only output a DVI signal, not a "real" HDMI signal. So some meta data will be missing.
Time Sleuth ignores any EDID sent from the sink, but maybe the Decimator adds HDMI meta data to the signal?
The deviation of the clock should be well within the allowed +/- 0.5%.
I always verify, if the timings are correct on an HDMI analyser. Let me double check, if the signal is within specification.
Do you have a model number/manual for the video mixer used?

**Comment by philipstarkey** [comment]
Ah, I see. That would probably explain it. There was actually another item in the chain, a BlackMagic HDMI -> SDI microconverter (as the video mixer, a BlackMagic ATEM 1 M/E Production studio 4k, has mostly SDI inputs). This converter supports multiple resolutions/framerates but doesn't do standard conversions like the Decimator. My guess is that the microconverter falls over because it doesn't get an EDID...yep, [this](https://forum.blackmagicdesign.com/viewtopic.php?f=3&t=58423) pretty much confirms it!

That's ok. I'm happy to use the Decimator to make it work (I can do comparisons with two chained Decimators vs just one Decimator to determine how much latency is introduced by the first Decimator)
